garage/misc/instrument.py

```diff
@@ -408,27 +273,23 @@ def run_experiment(stub_method_call=None,
     :param periodic_sync_interval: Time interval between each periodic sync,
      in seconds.
     """
-    assert stub_method_call is not None or batch_tasks is not None, \
-        "Must provide at least either stub_method_call or batch_tasks"
+    assert method_call is not None or batch_tasks is not None, \
+        "Must provide at least either method_call or batch_tasks"
 
     if use_cloudpickle is None:
-        for maybe_stub in (batch_tasks or [stub_method_call]):
-            # decide mode
-            if isinstance(maybe_stub, StubBase):
-                use_cloudpickle = False
-            else:
-                assert hasattr(maybe_stub, '__call__')
-                use_cloudpickle = True
-                # ensure variant exists
-                if variant is None:
-                    variant = dict()
+        for task in (batch_tasks or [method_call]):
+            assert hasattr(task, '__call__')
```

Comment:
This should probably be 
```suggestion
            assert callable(task)
```
Either way is fine though, since it was already like this.